assets/src/edit-story/components/library/panes/pageLayouts/pageLayout.js

```diff
@@ -107,18 +117,24 @@ function PageLayout(props) {
     setIsConfirming(false);
   }, [onConfirm, setIsConfirming]);
 
-  const handleKeyUp = useCallback(
-    ({ key }) => {
-      if (key === 'Enter' && isActive) {
-        handleClick();
-      }
-    },
-    [handleClick, isActive]
-  );
+  useKeyDownEffect(ref, ['enter', 'space'], handleClick, [handleClick]);
 
   return (
     <>
-      <PageLayoutWrapper pageSize={pageSize} role="listitem">
+      <PageLayoutWrapper
+        pageSize={pageSize}
+        role="button"
+        ref={ref}
+        tabIndex={0}
+        onFocus={() => setIsActive(true)}
+        onBlur={() => setIsActive(false)}
+        onMouseEnter={() => setIsActive(true)}
+        onMouseLeave={() => setIsActive(false)}
```

Comment:
🤔 Do we need to worry about re-renders here since these functions will be referentially different on every render? Should these be wrapped in `useCallback`?

Seems like this was just moved but maybe something we should think about in the future